android/java/org/chromium/chrome/browser/settings/BraveAccountPreference.java

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.settings;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.util.AttributeSet;
+import android.widget.TextView;
+
+import androidx.preference.PreferenceViewHolder;
+
+import org.chromium.build.annotations.NullMarked;
+import org.chromium.build.annotations.Nullable;
+import org.chromium.chrome.R;
+import org.chromium.components.browser_ui.settings.ChromeBasePreference;
+
+/** A preference for Brave Account that supports customizing title and summary text colors. */
+@NullMarked
+public class BraveAccountPreference extends ChromeBasePreference {
+    private final int mTitleTextColor;
+    private final int mSummaryTextColor;
+
+    public BraveAccountPreference(Context context, @Nullable AttributeSet attrs) {
+        super(context, attrs);
+
+        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.brave_account_preference);
```

Comment:
null check on `attrs`? As `attrs` marked as `Nullable`

Reply:
Probably not needed, as `obtainStyledAttributes()` should handle `null` gracefully as per its [documentation](https://developer.android.com/reference/android/content/Context#obtainStyledAttributes(android.util.AttributeSet,%20int[])):
> `set` | `AttributeSet`: This value may be `null`.

`ChromeBasePreference` has the same signature and handles `attrs` in the same way.